interface.go

```diff
@@ -16,33 +17,49 @@ type TableInfo struct {
 }
 
 type ColumnDef struct {
-	Path string    `json:"path"`
+	Name string    `json:"name"`
```

Comment:
What is the difference between `Name` and `Key`

Reply:
They are the same thing. Replacing Key with Name.